package.json

```diff
@@ -3,28 +3,28 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
-    "@adobe/aio-lib-core-logging": "^2.0.0",
     "@adobe/aio-sdk": "^3.0.0",
-    "@adobe/exc-app": "^0.2.17",
-    "cloudevents": "^4.0.2",
     "node-fetch": "^2.6.0",
+    "cloudevents": "^4.0.2",
+    "uuid": "^8.0.0",
+    "core-js": "^3.6.4",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-router-dom": "^5.2.0",
+    "react-error-boundary": "^1.2.5",
     "regenerator-runtime": "^0.13.5",
-    "uuid": "^8.0.0"
+    "@adobe/exc-app": "^0.2.21",
+    "@adobe/react-spectrum": "^3.4.0",
+    "@spectrum-icons/workflow": "^3.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.21.3",
-    "@babel/eslint-parser": "^7.21.3",
+    "jest": "^27.2.4",
     "@openwhisk/wskdebug": "^1.3.0",
-    "eslint": "^8.36.0",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-standard": "^17.0.0",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-jest": "^26.5.3",
-    "eslint-plugin-jsdoc": "^39.3.3",
-    "eslint-plugin-n": "^15.2.3",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^6.0.0",
-    "jest": "^27.2.4"
+    "@adobe/aio-lib-state": "2.1.1",
+    "@babel/core": "^7.8.7",
+    "@babel/polyfill": "^7.8.7",
+    "@babel/preset-env": "^7.8.7",
+    "@babel/plugin-transform-react-jsx": "^7.8.3"
```

Comment:
Is the polyfill, preset-env etc. used by something? could you cleanup package.json and remove the libs that are not used?